src/components/Dashboard/Dashboard.tsx

```
@@ -0,0 +1,362 @@
+      setIsModalOpen(true);
+  };
+
+  if (isHome) return <Home onStart={() => setIsHome(false)} />;
+
+  return (
+    <div className="min-h-screen bg-gray-50 dark:bg-gray-950 text-gray-900 dark:text-gray-100 transition-colors duration-300">
+      <header className="bg-white dark:bg-gray-900 border-b border-gray-200 dark:border-gray-800 sticky top-0 z-20">
+        <div className="max-w-7xl mx-auto px-4 h-20 flex flex-col justify-center">
+          <div className="flex items-center justify-between">
+            <div className="flex items-center space-x-4 shrink-0">
+              <button onClick={() => setIsHome(true)} className="p-2 hover:bg-gray-100 dark:hover:bg-gray-800 rounded-lg text-gray-500"><ChevronLeft className="w-5 h-5" /></button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)